packages/core/docs/integrate/integration-guide.md

```diff
@@ -394,16 +389,22 @@ export { cartGetters }
 
 In the default theme, components used to display or modify integration-specific data, like forms or checkout information are blank. This is because each integration might use different data formats and properties.
 
-You need to create a couple of Vue components:
+You need to create few Vue components and JavaScript files:
 
 | Component                                    | Props                               | Emits event |
 |----------------------------------------------|-------------------------------------|-------------|
 | components/UserBillingAddress.vue            | { address: Object }                 |             |
 | components/UserShippingAddress.vue           | { address: Object }                 |             |
+| components/Checkout/CartPreview.vue          |                                     |             |
+| components/Checkout/VsfPaymentProvider.vue   |                                     |             |
+| components/Checkout/VsfShippingProvider.vue  |                                     |             |
```

Comment:
these two are providers exposed by a specific integration library, that's not the theme thing

Reply:
While CT doesn't provide these functionalities, other providers might. I'd keep it as it is, or mark it as optional?